tests/runner.py

```diff
@@ -1761,16 +1761,19 @@ def skip_requested_tests(args, modules):
       which = [arg.split('skip:')[1]]
 
       print(','.join(which), file=sys.stderr)
+      skipped = False
       for test in which:
         print('will skip "%s"' % test, file=sys.stderr)
         suite_name, test_name = test.split('.')
         for m in modules:
           try:
             suite = getattr(m, suite_name)
             setattr(suite, test_name, lambda s: s.skipTest("requested to be skipped"))
+            skipped = True
             break
           except AttributeError:
             pass
```

Comment:
Seems like this might be better written as: 

```
suite = getattr(m, suite_name, None)
if suite:
   setattr(suite, test_name, lambda s: s.skipTest("requested to be skipped"))
￼   skipped = True
￼   break
```

Rather then a try/catch... but unrelated this change I guess.

Reply:
The windows and mac circleci bots didn't seem to be running, so I took the opportunity to both make this change and try to get them going.